blob/helper.go

```diff
@@ -4,29 +4,27 @@ import (
 	"bytes"
 	"sort"
 
-	"github.com/tendermint/tendermint/types"
-
-	"github.com/celestiaorg/celestia-app/pkg/shares"
-	apptypes "github.com/celestiaorg/celestia-app/x/blob/types"
+	squareblob "github.com/celestiaorg/go-square/blob"
+	"github.com/celestiaorg/go-square/shares"
 
 	"github.com/celestiaorg/celestia-node/share"
 )
 
 // BlobsToShares accepts blobs and convert them to the Shares.
-func BlobsToShares(blobs ...*Blob) ([]share.Share, error) {
-	b := make([]types.Blob, len(blobs))
-	for i, blob := range blobs {
-		namespace := blob.Namespace()
-		b[i] = types.Blob{
-			NamespaceVersion: namespace[0],
-			NamespaceID:      namespace[1:],
-			Data:             blob.Data,
-			ShareVersion:     uint8(blob.ShareVersion),
+func BlobsToShares(nodeBlobs ...*Blob) ([]share.Share, error) {
+	b := make([]*squareblob.Blob, len(nodeBlobs))
+	for i, nodeBlob := range nodeBlobs {
+		namespace := nodeBlob.Namespace()
+		b[i] = &squareblob.Blob{
+			NamespaceVersion: uint32(namespace[0]),
+			NamespaceId:      namespace[1:],
+			Data:             nodeBlob.Data,
+			ShareVersion:     nodeBlob.ShareVersion,
 		}
 	}
 
 	sort.Slice(b, func(i, j int) bool {
-		val := bytes.Compare(b[i].NamespaceID, b[j].NamespaceID)
+		val := bytes.Compare(b[i].Namespace().Bytes(), b[j].Namespace().Bytes())
```

Comment:
AFAIR we've `Namespace.Compare` method, isn't it?

Reply:
Sure, resolved via 96e1bcd3a862a815d6b35529219930471d18aac6